config.yml

```diff
@@ -440,8 +440,6 @@ tezosqa:
   "0.1":
     template_vars:
       BASE_IMAGE_VERSION: 3.8
-    build_args:
-      SMARTPY_VERSION: dev-20201025-47d8831b87b9b38626939be9e23cbff966f4499a
```

Comment:
Why?

Reply:
It failed yesterday https://github.com/ekino/docker-buildbox/runs/1460861085 
but today it seems ok https://smartpy.io/dev-20201025-47d8831b87b9b38626939be9e23cbff966f4499a/cli/install.sh

anyway, thanks for your comment, I repushed with `local-install-dev` instead of `local-install-auto` 

because tezos is moving fast, having the last dev version is better for dev experience (dev is very stable)

Reply:
but indeed it can be improved, we could even have all versions in the box

Reply:
just retried and it failed again https://github.com/ekino/docker-buildbox/actions/runs/386735040

Reply:
sorry this one too failed but it worked this morning :cry: https://github.com/ekino/docker-buildbox/actions/runs/386668130
will check everything and repush

Reply:
seen with SmartPy developpers (I should have read the script more attentively)

I re-pushed, installed version is:
```
root@4c9acde601da:/# /opt/SmartPy/SmartPy.sh --version
Version "dev", b5736653-dirty, Wed, 14 Oct 2020 14:40:48 +0000.
```

Reply:
following https://smartpy.io/releases.html looks like you can select a specific version like this for instance: https://smartpy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/install.sh

Reply:
didn't see that! but unfortunately it must be the release of the web IDE, because it doesn't change the installed cli:

```
root@a922f044b36b:/# curl -s https://smartpy.io/cli/install.sh | bash -s local-install-custom https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli /opt/SmartPy
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/smartpy.py to /opt/SmartPy/smartpy.py ...
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/smartpyio.py to /opt/SmartPy/smartpyio.py ...
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/smartpy_michelson.py to /opt/SmartPy/smartpy_michelson.py ...
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/smartpy_cli.py to /opt/SmartPy/smartpy_cli.py ...
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/browser.py to /opt/SmartPy/browser.py ...
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/version.py to /opt/SmartPy/version.py ...
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/SmartPy.sh to /opt/SmartPy/SmartPy.sh ...
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/install.sh to /opt/SmartPy/install.sh ...
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/smartml-cli.js to /opt/SmartPy/smartml-cli.js ...
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/reference.html to /opt/SmartPy/reference.html ...
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/asciidoctor.css to /opt/SmartPy/asciidoctor.css ...
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/coderay-asciidoctor.css to /opt/SmartPy/coderay-asciidoctor.css ...
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/scripts/demo.py to /opt/SmartPy/scripts/demo.py ...
[SmartPyInstaller] Downloading https://SmartPy.io/releases/20201126-184c26c49ed49bf4167d8c9dbbc3b56374e51aa0/cli/templates/welcome.py to /opt/SmartPy/templates/welcome.py ...
Wrote to /opt/SmartPy/package.json:

{
  "name": "SmartPy",
  "version": "1.0.0",
  "description": "",
  "main": "smartml-cli.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC"
}


npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN SmartPy@1.0.0 No description
npm WARN SmartPy@1.0.0 No repository field.

+ chalk@4.1.0
+ bs58check@2.1.2
+ libsodium-wrappers-sumo@0.7.8
added 22 packages from 12 contributors and audited 22 packages in 2.838s

3 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

[SmartPyInstaller] Installation successful in /opt/SmartPy
root@a922f044b36b:/# /opt/SmartPy/SmartPy.sh --version
Version "dev", b5736653-dirty, Wed, 14 Oct 2020 14:40:48 +0000.
```

Reply:
ping @Bladrak

Reply:
I agree with @mremi we need to set the installed version, simply because the install command is subject to changes (as shown in the PR), and the build must be reproductible. Even if the version changes frequently, we can still only edit one image per month for instance.

Reply:
ok, thanks repushed !